examples/static_react_task/webapp/cypress/e2e/static_react_task.cy.js

```diff
@@ -1,32 +1,38 @@
-describe("Loads static_react_task", () => {
+const pathToAssignmentId = "cypress/assignment_id.txt";
+
+describe("Requests", () => {
   it("Makes request for agent", () => {
     cy.intercept({ pathname: "/request_agent" }).as("agentRequest");
-    cy.visit("/");
+    cy.readFile(pathToAssignmentId).then((assignmentId) => {
+      cy.visit("/?worker_id=x&assignment_id=" + assignmentId);
+    });
```

Comment:
For requests, consist test results seem to occur when the intercept is before the visit statement. This is why this test has to be in its own describe block that does not have the beforeEach function in it.